rest/rest-resources/src/main/java/org/apache/brooklyn/rest/transform/CatalogTransformer.java

```diff
@@ -131,7 +132,11 @@ public static CatalogItemSummary catalogItemSummary(BrooklynRestResourceUtils b,
     }
 
     public static CatalogPolicySummary catalogPolicySummary(BrooklynRestResourceUtils b, CatalogItem<? extends Policy,PolicySpec<?>> item, UriBuilder ub) {
-        Set<PolicyConfigSummary> config = ImmutableSet.of();
+        final Set<PolicyConfigSummary> config = new HashSet<>();
+        final PolicySpec<?> spec = (PolicySpec<?>) b.getCatalog().createSpec((CatalogItem) item);
+        for (final SpecParameter<?> input : spec.getParameters()){
```

Comment:
Parameters are already ordered and you lose the ordering with the `HashSet`. Suggest to use `LinkedHashSet`. (for the case of = priority).
The API doesn't actually guarantee ordering (it's a `Set`), but Javascript will keep it anyway. Decided to keep it like this (for the entity case) and not introduce backward compatibility issues until needed.